.gitignore

```diff
@@ -18,10 +18,13 @@ __pycache__/
 /nginx/.htpasswd
 /nginx/htpasswd
 /nginx/Dockerfile
+/nginx/Dockerfile-nginx
```

Comment:
I'd like to see all the Dockerfiles in one folder (the project root or a dockerfiles folder), however if this is too much work to fix leave it as-is

When working with the old mstealy build it was a nightmare tracking down all the files

Reply:
actually leave this alone. I see you are using the relative path context in the docker-compose yml